### ISSUE TITLE
Is R > 4.4.0 really a requirement?

### ISSUE THREAD

**Issue by onurcanbektas** [compbiomed/BatchQC#99]
Is R > 4.4.0 really a requirement?
My of the HPC systems are still at much earlier version of R and can't really use BatchQC at the moment because of this.

**Comment by jessmcc22** [comment]
Yes, to use the latest methods in BatchQC, you do need to have the latest version of R. BatchQC is hosted on BioConductor, which requires using the most up-to-date R when adding updates to a package (this ensures compatibility among all dependencies within BioConductor tools and other R packages). You can choose to use an older version of BioConductor (and thus an older version of BatchQC), but you may not have access to all functionality (See News for a list of more recent additions). You will need a minimum of R version 3.5 to run the original BatchQC (which is hosted at https://github.com/mani2012/BatchQC), but everything hosted on this GitHub requires R 4.4 or higher. Having an outdated R is not just a BatchQC problem; therefore, we would recommend reaching out to your HPC manager and requesting an updated R (or inquiring if there is a way for you to host a local updated R) so that you have access to updated tools (including beyond BatchQC).
